### ISSUE TITLE
Source Control for  large FHIR resources: CodeSystems, ConceptMaps and ValueSets

### ISSUE THREAD

**Issue by chrisroederucdenver** [HOT-Ecosystem/hapi-fhir-jpaserver-starter#42]
"Deposit final CodeSystem Resource in a stable location"
It seems like GitHub would be nice for this, but it has file size limitations that a CodeSystem might violate. Has lfs (large file support) been looked into? How big a file can you put into Google Drive? Would we need to consider AWS buckets or the GCP (google cloud platform) equivalent?

**Comment by joeflack4** [comment]
@chrisroederucdenver Indeed, thanks Chris! Yes, @Sigfried  and I are doing that exact thing for TermHub. I just need to finish setting up Git LFS. I think we could share this repository between TermHub and the TIMS work.

The way we're using it now is:
- [TermHub](https://github.com/jhu-bids/TermHub): the web app that uses the content
- [TermHub ConceptSets](https://github.com/jhu-bids/termhub-csets/tree/9589f29fb66a38e3de3fa937e7dfde8849dee889) git submodule
- [TermHub vocab](https://github.com/jhu-bids/termhub-vocab/tree/a1a568eac094f4ee3ed947098de4bb10d8c98842) git submodule (empty thus far)

We could probably rename to "TIMS csets" and "TIMS vocab", etc.

**Comment by chrisroederucdenver** [comment]
So, Joe. LFS works for vocabulary: it can handle the file size and version them too? how big a file have you put in there.

**Comment by joeflack4** [comment]
Yep! [Looks like](https://docs.github.com/en/repositories/working-with-files/managing-large-files/about-git-large-file-storage) the free offering supports 2GB. I think there is just _one_ dataset that Siggie and I are working with from N3C that is bigger than that, but probably we would be better off splitting it into two files rather than upgrading to the paid version to support higher sizes.

Regarding versioning, yep, it supports that.

**Comment by chrisroederucdenver** [comment]
We might find a use for how expensive the paid version is.
